client/src/App.css

```diff
@@ -29,16 +28,27 @@
   color: white;
 }
 
-.halogen-line{
-height: 3px;
-
-
+.halogen-line {
+  height: 3px;
 }
 
 .App-link {
   color: #1f657a;
 }
 
+.lightbox-sub {
```

Comment:
There's a tricky issue here with `SketchPicker` and it's parent element `.lightbox-sub`.

Since the color-picker itself is absolutely positioned with a property of `top-80`, that means that the element will position itself 80 pixels from the top of the **nearest positioned element**. That's the tricky part. `.lightbox-sub` here is not _explicitly_ positioned in the document. Meaning that `top-80` will be referencing the nearest parent element that IS positioned (probably the body element).

By adding `position: relative` to `.lightbox-sub`, the color picker will then be using `.lightbox-sub` as it's frame of reference for `top-80`,

This might help explain it a little better - https://www.loom.com/share/a76119b48d4e4acdb1f983a86a567a96